extensions/Persona/Config.pm

```diff
@@ -19,11 +19,6 @@ use constant REQUIRED_MODULES => [
         module  => 'JSON',
         version => 0,
     },
-    {
-        package => 'libwww-perl',
-        module  => 'LWP::UserAgent',
-        version => 0,
-    },
```

Comment:
Can we just remove extensions/Persona from BMO, or is this not BrowserID Persona?